src/fmu/sumo/explorer/objects/surface.py

```diff
@@ -13,7 +14,7 @@ def __init__(self, sumo: SumoClient, metadata: Dict) -> None:
             sumo (SumoClient): connection to Sumo
             metadata (dict): surface metadata
         """
-        super().__init__(sumo, metadata)
+        Child.__init__(self, sumo, metadata)
```

Comment:
Is this the preferred way of calling parent methods?

Reply:
Noo, the correct solution is to remove __init__ altogether because it will be run implicitly when you inherit from child without adding anything. My bad, I reacted to a pylint code, and didn't properly understand what the issue was.